src/github/pullRequestOverview.ts

```diff
@@ -585,6 +585,8 @@ export class PullRequestOverviewPanel extends IssueOverviewPanel {
 				review: review,
 				reviewers: this._existingReviewers
 			});
+			//just making sure that pr review status is approved on git servers
```

Comment:
I think this timeout isn't necessary, I believe GitHub will return the correct information immediately. Let me know if you've seen different behavior

Reply:
When I was testing I could see some inconsistency with refreshing the list/view.  After I added ```timeout``` there wasn't in any inconsistency.

Reply:
Let me know if you want me to remove the ```timeout```

Reply:
I have removed the ```timeout```.